site/content/docs/user/known-issues.md

```diff
@@ -40,26 +40,26 @@ This problem is related to a bug in [docker on macOS][for-mac#3663]
 
 If you see something like the following error message:
 ```bash
```

Comment:
Why this is not.changed to codefromline too?

Reply:
Code from inline is copyable things to save or run.

This is sable output

Reply:
This is not something you should copy and paste